app/components/Home/Yaroslav_Grushchak/Projects/Project_1/App.css

```diff
@@ -0,0 +1,95 @@
+.App {
```

Comment:
Read about css preprocessor stylus, and use files with ext .styl instead of .css

Reply:
Done.